.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
```

Comment:
Should this be `merge`?

Reply:
Don't think so. ref

https://github.com/orgs/community/discussions/26724
https://stackoverflow.com/questions/60710209/trigger-github-actions-only-when-pr-is-merged